internal/selfmonitor/config/config_builder.go

```diff
@@ -112,6 +112,10 @@ func makeScrapeConfig(scrapeNamespace string) []ScrapeConfig {
 					Regex:        "otelcol_.+;.+/([a-zA-Z0-9-]+)",
 					TargetLabel:  "pipeline_name",
 				},
+				{
+					Action: LabelDrop,
+					Regex:  "^(receiver|exporter|directory|service_instance_id|service_version|transport|service_name|alertname|job|instance|namespace|node)$",
+				},
```

Comment:
Will it drop the exporter label after the rule above is applied (which introduces a pipleine_name label)?

Reply:
yes, label pipeline_name will stay others removed